pandas/_libs/src/ujson/python/objToJSON.c

```diff
@@ -509,7 +501,7 @@ static void *PyTimeToJSON(JSOBJ _obj, JSONTypeContext *tc, void *outValue,
     GET_TC(tc)->newObj = str;
 
     *outLen = PyBytes_GET_SIZE(str);
-    outValue = (void *)PyBytes_AS_STRING(str);
+    char *outValue = PyBytes_AS_STRING(str);
```

Comment:
this looks like it should have implications for the return type of this function.  is the returned outValue ever used?

Reply:
In the scope of this function the current assignment to `outValue` goes nowhere. Particularly for strings it has no use, as `Object_getString` ignores it altogether. Definitely confusing

Goal in the next PR(s) is to split out all of these functions that return void pointers into functions that return either `char *` or int to streamline things and make readable

Reply:
Thanks.  Pending green, LGTM.